confluence-syntax-xhtml/src/main/java/org/xwiki/contrib/confluence/parser/xhtml/internal/wikimodel/ConfluenceXWikiGeneratorListener.java

```diff
@@ -158,15 +158,7 @@ public void onReference(WikiReference reference)
                 }
 
                 resourceReference = attachmentResourceReference;
-            } else if (confluenceReference.getUser() != null) {
-                UserResourceReference userResourceReference = new UserResourceReference(confluenceReference.getUser());
-
-                if (confluenceReference.getAnchor() != null) {
-                    userResourceReference.setAnchor(confluenceReference.getAnchor());
-                }
-
-                resourceReference = userResourceReference;
-            } else if (confluenceReference.getAnchor() != null) {
```

Comment:
Since the `<ri:user>` tag can't be a child to a `<ac:link ac:anchor>` tag, this piece of code is not needed.

Reply:
Would make pull requests easier to review without totally unrelated changes in it :)

At least it's a different commit.

Reply:
Sorry. The laziness got the better of me. I wanted to hit two issues with one PR and call it a day :P. (implement the issue and also fix the build errors)